.golangci.yml

```diff
@@ -83,6 +83,7 @@ linters-settings:
       - name: blank-imports
       - name: indent-error-flow
       - name: unreachable-code
+      - name: redefines-builtin-id
```

Comment:
Hi, @Affan-7 , To avoid multiple PR conflicts, you can add this and fix it locally. The modifications here will be submitted together in the end.

Reply:
I just rebased this PR.